Public/Src/Cache/ContentStore/Distributed/NuCache/BlobCentralStorage.cs

```diff
@@ -16,7 +15,9 @@
 using BuildXL.Utilities.Tasks;
 using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.Blob;
+using DateTimeUtilities = BuildXL.Cache.ContentStore.Utils.DateTimeUtilities;

```

Comment:
Why?

Reply:
Because the helper's functionality has moved from one type to another.

---
In reply to: [283070386](https://github.com/microsoft/BuildXL/pull/289#discussion_r283070386) [](ancestors = 283070386)